### ISSUE TITLE
[BUG] The IC and Ghost tabs are missing from the top-right of the screen

### ISSUE THREAD

**Issue by Eskjjlj** [voidcrew/Voidcrew-LRP#258]
## Round ID:
Every round. I first noticed it one or two weeks ago.

## Testmerges:
Doesn't matter.

## Reproduction:
I log on the server and my tabs are missing. Restarting byond, deleting its cache, updating to a newer version or deleting IE cache did not solve the issue. A friend told me they had the same bug.
![unknown](https://user-images.githubusercontent.com/45199549/154839619-bd0d04e8-917e-4c96-81d5-765fc935af03.png)

**Comment by Sbmhawk** [comment]
The "Object" tab is also missing

**Comment by SuperNovaa41** [comment]
yeah i've seen this a few times, and i have no fucking idea why its happening
